nrf_modem/Kconfig

```diff
@@ -15,12 +15,15 @@ choice NRF_MODEM_BUILD_STRATEGY
 	depends on NRF_MODEM
 	prompt "Build strategy"
 
-config NRF_MODEM_LINK_BINARY
-	bool "Link binary"
+config NRF_MODEM_LINK_BINARY_LTE
+	bool "Link LTE binary"
+
+config NRF_MODEM_LINK_BINARY_DECT_PHY
+	bool "Link DECT binary"
```

Comment:
Perhaps `DECT PHY` is more correct here?

Reply:
Renamed in the second commit (lte->cellular rename commit). Maybe it should be in the first.